cmd/zetacored/parse_genesis.go

```diff
@@ -172,6 +177,18 @@ func ImportDataIntoFile(
 	return nil
 }
 
+func ModifyStakingState(
+	importAppState map[string]json.RawMessage,
+	cdc codec.Codec) error {
+	importedStakingGenState := stakingtypes.GetGenesisStateFromAppState(cdc, importAppState)
+	fmt.Println("Validators :", len(importedStakingGenState.Validators))
+	fmt.Println("Delegations :", len(importedStakingGenState.Delegations))
+	fmt.Println("UnbondingDelegations :", len(importedStakingGenState.UnbondingDelegations))
+	fmt.Println("Redelegations :", len(importedStakingGenState.Redelegations))
+	fmt.Println("LastValidatorPower :", len(importedStakingGenState.LastValidatorPowers))
```

Comment:
We should rather have logs in the high level function, are these logs even necessary?